frontend/public/components/resource-quota.jsx

```diff
@@ -1,38 +1,30 @@
+import * as _ from 'lodash-es';
 import * as React from 'react';
 
-import { ColHead, DetailsPage, List, ListHeader, ListPage } from './factory';
+import { ColHead, DetailsPage, List, ListHeader, MultiListPage } from './factory';
 import { Cog, SectionHeading, navFactory, ResourceCog, ResourceLink, ResourceSummary } from './utils';
-import { registerTemplate } from '../yaml-templates';
-
-registerTemplate('v1.ResourceQuota', `apiVersion: v1
-kind: ResourceQuota
-metadata:
-  name: example
-spec:
-  hard:
-    pods: "4"
-    requests.cpu: "1"
-    requests.memory: 1Gi
-    limits.cpu: "2"
-    limits.memory: 2Gi
-`);
-
 
 const menuActions = [Cog.factory.ModifyLabels, Cog.factory.ModifyAnnotations, Cog.factory.Edit, Cog.factory.Delete];
 
+export const QuotaName = ({quota: quota}) => {
+  const kind = quota.metadata.namespace ? 'ResourceQuota' : 'quota.openshift.io:v1:ClusterResourceQuota';
+  return <React.Fragment>
+    <ResourceCog actions={menuActions} kind={kind} resource={quota} />
+    <ResourceLink kind={kind} name={quota.metadata.name} namespace={quota.metadata.namespace} className="co-resource-link__resource-name" />
```

Comment:
I believe you can make this `kind={quota.kind}` if you add a model to

https://github.com/openshift/console/blob/master/frontend/public/models/index.ts

Then you wouldn't need the special component for name.